Task8/Task8/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение аргумента функции:");
+             var x  = double.Parse(Console.ReadLine());
+
+            Console.WriteLine($"f({x}) = {MyFunction(x)}");
+
+            Console.ReadKey();
+        }
+
+        static double MyFunction(double x)
+        {
+            if (x > 0)
+                return Math.Exp(-x); 
+            else if (x > -1 && x <= 0)
```

Comment:
Здесь сравнение с 0 лишнее.
```suggestion
            else if (x > -1)
```